packages/features/insights/filters/index.tsx

```diff
@@ -51,7 +61,9 @@ export const Filters = ({ showRoutingFilters = false }: { showRoutingFilters?: b
           <>
             <RoutingFormFilterList />
             <BookingStatusFilter />
-            <RoutingFormFieldFilter />
+            {routingFormFieldIds.map((fieldId) => {
```

Comment:
I had to take this approach since I was getting a `Rendered more hooks than during the previous render` error trying to map the routing form field filters inside of `<RoutingFormFieldFilter />`